coala_quickstart/generation/Settings.py

```diff
@@ -61,7 +61,7 @@ def generate_ignore_field(project_dir, languages, extset, ignore_globs):
     for glob in ignore_globs:
         gitignore_files = {file
                            for file in collect_files([glob], null_printer)}
-        if not all_files.isdisjoint(gitignore_files):
+        if all_files.isdisjoint(gitignore_files):
```

Comment:
Yeah, that's really confusing to me as well. But won't all_files will always be disjoint from gitignore_files due to this https://github.com/hemangsk/coala-quickstart/blob/614598959e007d659454025cc22ccf03f4b85714/coala_quickstart/generation/Settings.py#L58